TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Knative Trademark committee"
+linkTitle: "Trademark committee"
+weight: 50
+type: "docs"
+aliases:
+  - /contributing/trademark-committee/
+---
+
+The Knative Trademark Committee (KTC)
+[advises the trademark owner on the usage and enforcement of the Knative mark](./GOVERNANCE.md).
+It is a separate body from the [Steering Committee](./STEERING-COMMITTEE.md) to
+better represent the needs of vendors and other companies which market products
+in conjuction with the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change. We expect over time we will adapt
+the way we run this committee, based on feedback from the community.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve the scope and definition of additional derivative Knative
+   marks (e.g. Knative Serving, Knative Core, etc). It is expected that the
+   Trademark Committee will delegate one or more marks to the steering committee
+   to provide branding for nascent efforts which are not yet subject to
+   conformance and third-party licensing.
```

Comment:
We should take the 'It is expected ...' and put it in delegated authority. I'll put a comment down there -- similar to my previous comment we can focus the charter on simply stating the powers, and use other sections to clarify implementation.

Reply:
I think we all identified on Tuesday that it was important that there is some usage of the "Knative" name that can be used by experiments and so forth, so I wanted to put that expectation (constraint, rather than power) into the charter somewhere. I can move it out of the bullet points if that helps.